### PR TITLE
Cloud 175 monitoring split

### PR DESCRIPTION
**Pull request by patrickbirch**
Split install/Monitoring.md into sections.

### CODE REVIEW

File: install/monitoring.md [hunk 1]
```diff
@@ -25,7 +27,12 @@ Following steps are needed to install both PMM Client and PMM Server preconfigur
    * `credentials.username` should correspond to the `serverUser` key in the `pmm` section of the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.
    * `credentials.password` should correspond to a value of the `pmmserver` secret key specified in `deploy/secrets.yaml` secrets file. Note that password specified in this example is the default development mode password not intended to be used on production systems.
 
-4. Now the PMM is installed, and it is time to update ``pmm`` section in the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.
+   ### Installing the PMM Client
```
Comment (fiowro) [inline install/monitoring.md]:
```suggestion
### Installing the PMM Client
```

File: install/monitoring.md [hunk 2]
```diff
@@ -25,7 +27,12 @@ Following steps are needed to install both PMM Client and PMM Server preconfigur
...
+
+
+   The following steps initiate the PMM client installation:
```
Comment (fiowro) [inline install/monitoring.md]:
```suggestion
   The following steps are needed for the PMM client installation:
```

File: install/monitoring.md [hunk 3]
```diff
@@ -25,7 +27,12 @@ Following steps are needed to install both PMM Client and PMM Server preconfigur
...
+
+1. The PMM client is initiated by updating the ``pmm`` section in the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.
```
Comment (fiowro) [inline install/monitoring.md]:
```suggestion
1. The PMM client installation is initiated by updating the ``pmm`` section in the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.
```

File: install/monitoring.md [hunk 4]
```diff
@@ -3,7 +3,9 @@ Monitoring
 
 The Percona Monitoring and Management (PMM) [provides an excellent solution](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/manual/monitoring.html#using-pmm) to monitor Percona XtraDB Cluster.
 
-Following steps are needed to install both PMM Client and PMM Server preconfigured to monitor Percona XtraDB Cluster on Kubernetes or OpenShift.
+### Installing the PMM Server
+
+Following steps are needed to install the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift.
```
Comment (delgod) [inline install/monitoring.md]: It is needed to mention that -- This step is optional If you already have PMM Server installed (inside or outside of Kubernetes)

File: install/monitoring.md [hunk 5]
```diff
@@ -3,7 +3,9 @@ Monitoring
 
 The Percona Monitoring and Management (PMM) [provides an excellent solution](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/manual/monitoring.html#using-pmm) to monitor Percona XtraDB Cluster.
 
-Following steps are needed to install both PMM Client and PMM Server preconfigured to monitor Percona XtraDB Cluster on Kubernetes or OpenShift.
+### Installing the PMM Server
+
+This first section installs the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift. The following steps are optional if you already have installed the PMM Server. The PMM Server is available on your network and does not require another installation in Kubernetes.
```
Comment (fiowro) [inline install/monitoring.md]:
```suggestion
This first section installs the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift. The following steps are optional if you already have installed the PMM Server. The PMM Server available on your network does not require another installation in Kubernetes.
```

**Pull request merged**